consensus/consensus.md

```diff
@@ -134,11 +134,14 @@ nonce:
 #### Key Blocks
 
 ```
-PROTOCOL_VERSION: 2
-GENESIS_VERSION: 2
+PROTOCOL_VERSION: 21
+GENESIS_VERSION: 21
 ```
 
-- the timestamp of a block MUST be smaller than `now() + 30m` TODO: re-calc for NG
+- the timestamp of a key block MUST be smaller than `now() + 10m`
+- the timestamp of a key block MUST be bigger than:
+  - the median timestamp of the last 11 blocks (if height >= 12)
```

Comment:
I see that for block height 12 the median computation excludes genesis (that has height 0). Ok.